udapi/core/node.py

```diff
@@ -253,6 +252,10 @@ def parent(self, new_parent):
     def children(self):
         return self._children
 
+    @children.setter
```

Comment:
Should this be a public method? How to asses the ordering of the children?
The standard way is to use set_parent.